packages/components/src/components/canvas/canvas.tsx

```diff
@@ -1,6 +1,7 @@
 import type { PropTypes } from '../../types';
 import { AllowedTypes, attachComponentCss, getPrefixedTagNames, validateProps } from '../../utils';
-import { Component, Element, EventEmitter, h, type JSX, Prop, Event, State, Host, Fragment } from '@stencil/core';
+import type { EventEmitter} from '@stencil/core';
+import { Component, Element, h, type JSX, Prop, Event, State, Host, Fragment } from '@stencil/core';
```

Comment:
You can also just add type in front of the imported type instead of a new import line
```suggestion
import { Component, Element, h, type JSX, Prop, Event, State, Host, Fragment, type EventEmitter } from '@stencil/core';
```